### ISSUE TITLE
can we include more information on the software versions used?

### ISSUE THREAD

**Issue by llavall** [SiLeBAT/FSK-Lab#208]
I would like to ask if we could include more information about the environment under which the model was created / excecuted. 
I have in mind something like that:
platform : Linux DEBIAN 9 (Stretch)
KNIME: Server v4.6.1
FSKlab build: de.bund.bfr.knime.fsklab.feature_1.0.0.201807061316
R:  4.4.4
R plugin mc2d : Version xx.x.x

I expect this would increase the transparency and we can later use these information in order to provide docker solution or something else to make sure an old model cann still be executed.

Best wishes
Lars

**Comment by mfilter** [comment]
very good idea. @llavall : please also check the list of suggestions for the metadata schema, as we would have to add that there as well.

**Comment by llavall** [comment]
This idea has been evolved to also provide a new tab in the FSK-outport. As suggested by Ahmad we could have a table like view which automatically compares the software/plugin version when creating the model and the present versions. In addition it could also change the colour if a difference was found. We hope that the user this way has an easy overview about all versions and on top a fast and simple comparision.

**Comment by mfilter** [comment]
Good idea.

**Comment by schuelet** [comment]
@mfilter Is this idea still good enough to keep the ticket open?

**Comment by mfilter** [comment]
Yes, I still like the idea. I suggest to find a pragmatic solution, e.g. we could probably add the information to the README.txt
Information on the R libraries versions is already captured.
